src/contract_wrappers/modules/sto/capped_sto_wrapper.ts

```diff
@@ -98,12 +98,15 @@ interface ChangeAllowBeneficialInvestmentsParams extends TxParams {
   allowBeneficialInvestments: boolean;
 }
 
-interface BuyTokensParams extends TxPayableParams {
+interface BuyTokensParams extends TxParams {
   beneficiary: string;
+  value: BigNumber;
+  from: string;
```

Comment:
`from` should not be mandatory. If it's not present, it is added by default by abi-wrappers

Reply:
if I make this change I get the next error:

> Argument of type 'string | undefined' is not assignable to parameter of type 'string'.\n  Type 'undefined' is not assignable to type 'string'

`assert.isETHAddressHex('beneficiary', params.from);`

In this case should I cast params.from as String?

Reply:
Yes, you should check if params.from is undefined first. You cloud use `=== undefined`